General/AudioProfile.vb

```diff
@@ -1123,12 +1123,19 @@ Public Class GUIAudioProfile
                         sb.Append(" -q:a " & CInt(Params.Quality))
                 End Select
             Case AudioCodec.AC3
+                If Not Params.CustomSwitches.Contains("-c:a ") Then
+                    sb.Append(" -c:a ac3")
```

Comment:
Is there a reason for setting the codec explicitly? ffmpeg does auto detect it based on the output file extension ac3. StaxRip usually sets things only of strictly necessary.

Reply:
For example, if you have an m4a file and want to convert it to ac3 without this note, you will get an error message. At least in my case.